Description: Correct elsa mysql_dir directory
 .
 securityonion-elsa-extras (20131117-1ubuntu0securityonion23) precise; urgency=low
 .
   * Remove old BRO_NOTICE pattern
Author: Scott Allen Runnels <srunnels@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- securityonion-elsa-extras-20131117.orig/bin/securityonion_elsa_register.rb
+++ securityonion-elsa-extras-20131117/bin/securityonion_elsa_register.rb
@@ -66,7 +66,7 @@ class ELSAConfig
               "interval"=> 0,
               "whitelist" => []
             },
-            "mysql_dir" => "/nsm/elsa/data/mysql",
+            "mysql_dir" => "/nsm/elsa/data/elsa/mysql",
             "stats" => {
               "retention_days" => 365
             },
